### DISCUSSION TITLE
How to log test runners steps in headless mode to console?

### DISCUSSION THREAD

**Discussion by Jan-Lubiejewski** [cypress-io/cypress#29409] [Questions and Help]
I want to run the test in headless mode but I want it to print the steps it's performing like in the headed mode in the GUI. But I want it to be printed in the headless mode, in console instead. How to do it?

**Comment by AlexGuironnetRTE** [comment]
Hello,

To do that we've been using the plugin [cypress-terminal-report ](https://github.com/archfz/cypress-terminal-report) for a while now, I don't know if that matches your needs?
